code/datums/quirks/negative_quirks/negative_quirks.dm

```diff
@@ -1535,3 +1535,18 @@
 	SIGNAL_HANDLER
 	quirk_holder.client.give_award(/datum/award/achievement/misc/debt_extinguished, quirk_holder)
 	UnregisterSignal(source, COMSIG_MOB_LOGIN)
+
+/datum/quirk/numb
+	name = "Numb"
+	desc = "You cannot feel pain at all."
+	icon = FA_ICON_STAR_OF_LIFE
+	quirk_flags = QUIRK_HUMAN_ONLY
+	value = -4
+	gain_text = null // handled by trauma
+	lose_text = null
+	medical_record_text = "Patient's body has badly desenvolved grown nervous system leading to complete inability to feel pain. "
```

Comment:
```suggestion
	medical_record_text = "Patient's body has a poorly developed nervous system, leading to a complete inability to feel pain. "
```
Not sure what this was originally meant to say, maybe this is better?

Reply:
Also, forgot to change that after remind myself 3 times to do it.

Thanks for the clearer-description.